xml/cap_depl_eirini.xml

```diff
@@ -78,8 +78,38 @@ env:
    </step>
    <step>
     <para>
-     Depending on your cluster configuration, Metrics Server need to be deployed
-     . Use &helm; to install the latest stable Metrics Server.
+     After initiating the <command>helm install</command> command to deploy
+     <literal>scf</literal>, monitor the status of the pods.
+    </para>
+<screen>&prompt.user;watch --color 'kubectl get pods --namespace scf'</screen>
+    <para>
+     In particular, take note of the <literal>secret-generation</literal> pod.
+     When its <literal>STATUS</literal> reaches a <literal>Running</literal>
+     state, a certificate signing request (CSR) from the
+     <literal>scf-bits-service-ssl-cert</literal> must be approved, Once
```

Comment:
I think we can just repro the text from Jan's helm notes here:
![image](https://user-images.githubusercontent.com/6218159/70263450-3b1d8d80-174b-11ea-8d6c-8ce20bce62fd.png)

Replace the template expression for the namespace with scf-namespace or something similar.